conf/etc/iem_ha.json

```diff
@@ -9,5 +9,17 @@
             "SearchKey": "member"
         }
     },
-    "resource": {}
-}
+    "resource": {
+        "resources": ["io_path_health-c1", "io_path_health-c2"],
+        "modules": {
+            "IOStop": {
+                "SearchKey": "stop",
+                "IEM": "IEC:WS0080030002:On $host Detected unhealthy IO services. Stopping IO services on $host server."
+            },
+            "IOStart": {
+                "SearchKey": "start",
+                "IEM": "IEC:IS0080030001:Starting IO services on $host server."
```

Comment:
`IO services from $otherHost are being started on $host.`

We can find out otherHost from suffix(c1, c2) of of resource names.

Reply:
Done